sopt/src/main/scala/dagr/sopt/cmdline/CommandLineException.scala

```diff
@@ -21,65 +21,66 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package dagr.core.cmdline
 
-private[core] object CommandLineException {
-  private[core] def format(msg: String, e: Exception): String = {
+package dagr.sopt.cmdline
+
+object CommandLineException {
+  def format(msg: String, e: Exception): String = {
     msg + ": " + e.getMessage + "\n" + e.getStackTrace.mkString("\n")
   }
 }
 
 /** Base class for all exceptions thrown by the command line parsing */
-private[core] class CommandLineException(msg: String) extends RuntimeException(msg) {
+class CommandLineException(msg: String) extends RuntimeException(msg) {
```

Comment:
This feels a bit cumbersome.  Why not make CommandLineException a bit more flexible:
```scala
class CommandLineException(msg: String, e: Option[Exception])
  extends RuntimeException(e.map(x => msg + ": " + x.getMessage + "\n" + x.getStackTrace.mkString("\n")) getOrElse msg)
```

Then each of the children can become:

```scala
class FooException(msg:String, e: Option[Exception]) extends CommandLineException(msg, e)
```